core/controllers/base.py

```diff
@@ -229,18 +229,18 @@ def __init__(self, request, response):  # pylint: disable=super-init-not-called
                             user_settings.last_logged_in)):
                     user_services.record_user_logged_in(self.user_id)
 
-        self.role = (
-            feconf.ROLE_ID_GUEST
-            if self.user_id is None else user_settings.role)
+        self.roles = (
+            [feconf.ROLE_ID_GUEST]
+            if self.user_id is None else user_settings.roles)
         self.user = user_services.get_user_actions_info(self.user_id)
 
         if not self._is_requested_path_currently_accessible_to_user():
             auth_services.destroy_auth_session(self.response)
             return
 
-        self.values['is_moderator'] = (
-            user_services.is_at_least_moderator(self.user_id))
-        self.values['is_admin'] = user_services.is_admin(self.user_id)
+        self.values['is_moderator'] = user_services.is_moderator(self.user_id)
+        self.values['is_curriculum_admin'] = (
```

Comment:
I would suggest checking to see where it's used. This may uncover cases where we're looking for an "admin"-type person but which no longer make sense with a "curriculum admin". Ideally we get rid of it completely.

Reply:
Looks like it's not been used anywhere in eth UI but we pass it in reponse for most of the handlers. I can do this cleanup if needed in a seperate PR! (Let me know if it's okay to do it ina  seperate PR then I'll create an issue!)
Few examples:
![image](https://user-images.githubusercontent.com/16653571/126416491-a3e0e9a8-5ee5-4618-82e9-bd61109a7c79.png)
![image](https://user-images.githubusercontent.com/16653571/126416508-1b29ce89-cc26-4d44-b3d4-16896a3836bb.png)

Note: We also send is_super_admin and is_moderator*

Reply:
I would prefer to drop it here, but if you would prefer a separate PR then please file an issue and add a TODO.